src/components/product/product.tsx

```diff
@@ -24,7 +26,17 @@ const Product: React.FC<ProductProps> = ({ title, description, price, imageUrl }
         className="text-md whitespace-nowrap overflow-hidden text-ellipsis"
         dangerouslySetInnerHTML={{ __html: sanitizedDescription }}
       />
-      <p className="text-gray-600">{price}</p>
+      <p className="text-neutral-600">{price}</p>
+      {onAddToCart && (
+        <Button
+          color='secondary'
+          className="h-full mt-2 py-3 flex justify-center items-center"
```

Comment:
Why the class `h-full`?

Please provide an example of screenshot of Storybook.

Reply:
![Screenshot from 2024-12-09 08-52-37](https://github.com/user-attachments/assets/0f8deffc-ea86-43bc-9331-d3c9b25db8fd)

Reply:
Why the class `h-full`?

Reply:
Its a mistake sir